backend/production-optimiser-api/src/main/java/it/polimi/productionoptimiserapi/dtos/OptimizationModelDTO.java

```diff
@@ -0,0 +1,31 @@
+package it.polimi.productionoptimiserapi.dtos;
+
+import it.polimi.productionoptimiserapi.entities.OptimizationModel;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import java.util.Set;
+import lombok.Builder;
+import lombok.Data;
+import org.hibernate.validator.constraints.URL;
+
+@Data
+@Builder
+public class OptimizationModelDTO {
+
+  @NotBlank(message = "name must be provided")
+  public String name;
+
+  @NotBlank(message = "apiUrl must be provided")
+  @URL(message = "apiUrl must be a valid URL")
+  public String apiUrl;
+
+  @NotEmpty(message = "Optimization model must be associated to at least one user")
```

Comment:
maybe this constraint is not needed, admin should be able to create model that will not be assigned to anybody at the moment

Reply:
Yes I think it makes sense to drop this constraint